src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -275,8 +273,6 @@ class TableView extends React.Component<TableViewProps> {
         field={field}
         location={location}
         eventView={eventView}
-        /* TODO(leedongwei): Verbosity is due to error in Prettier, fix after
```

Comment:
This comment is no longer relevant. So I deleted it. 

/cc @leedongwei

Reply:
I'm guessing that this was fixed by the change in type for `Metadata` that I did 2-3 weeks ago?

Reply:
Maybe.